### DISCUSSION TITLE
Question: Relating an endurant to a type by which it is categorized

### DISCUSSION THREAD

**Discussion by HuibertDeVries** [nemo-ufes/gufo#30] [Q&A] [answered]
I have two questions about the model below (picture and code block)

1. Content-wise:
We would like to indicate that a Person could fulfill a Organizational Role, but would not explicitly model all possible instances of Organizational Role as subclasses of Person.  
The advantage of this is that we leave unspecified which subclasses can end up being added. We only model the fact that there are different subclasses of Position following from the Role that Position has. 
Is this a 'correct' way to use gUFO?

7. Visually:
The property gufo:categorizes consideres Person and OrganizationalRole as instances of respectively gufo:Kind and gufo:Type.
The property :fulfillsTheRoleOf consideres Person and OrganizationalRole as classes.
How can we visualize efficiently the categorizes-relation between both classes without suggesting that instances of OrganizationalRole categorize instances of Person?

**Model**
![Categorizes](https://github.com/nemo-ufes/gufo/assets/122520770/08bb56d0-622d-4bf4-8852-9eef8972c038)

```
:Person a gufo:Kind ;

rdfs:subClassOf gufo:FunctionalComplex .

:OrganizationalRole a gufo:Type ;
	rdfs:subClassOf gufo:Role .

:OrganizationalRole gufo:categorizes :Person .

:fulfillsTheRoleOf a owl:ObjectProperty ;
	rdfs:domain ex:Person ;
	rdfs:range ex:OrganizationalRole .

:ProgramManager a :OrganizationalRole .

:John a :Person ;
	:fulfillsTheRoleOf :ProgramManager .
```

**Comment by jpalmeida** [comment, marked as answer]
Hi Huibert, to me your code is fine. It just does not capture that :fulfillsTheRoleOf corresponds to instantiation (which would have been capture directly with rdf:type, or given that this is contingent instantiation with [gufo:TemporaryInstantiationSituation](https://nemo-ufes.github.io/gufo/#TemporaryInstantiationSituation)).

**Comment by HuibertDeVries** [comment]
Hi Joao Paulo,
actually the relation is not meant to be an rdf:type.
The instances of Organizational Role are after all types.
So Persons instantiate classes that are instances of Organizational Role.
This way of modelling was meant to express this in a compact way, but I am not sure if it is a correct use of gUFO.
